apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/compute/functions/ImageToOperatingSystem.java

```diff
@@ -80,9 +80,9 @@ public OperatingSystem apply(final Image from) {
             is64Bit = matcher.group(2).equals("x64");
          }
       } else {
-         if (imageName.contains("Red Hat EL")) {
+         if (imageName.contains("Red Hat")) {
```

Comment:
If we do find out the reason (other than "this one seemed to work" ;-)), perhaps add a comment to explain?